v2/cli/init_command.go

```diff
@@ -0,0 +1,248 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"regexp"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/brigadecore/brigade/v2/internal/file"
+	"github.com/urfave/cli/v2"
+)
+
+type InitFields struct {
```

Comment:
Let's downcase the name of this type so it's not exported. There's probably no harm in it being exported, but as a general rule, keep things _unexported_ until there's a reason not to.